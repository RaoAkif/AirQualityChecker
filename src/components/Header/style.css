.header > img {
  width: 100%;
}

.date-time {
  position: absolute;
  border-radius: 50%;
  background-color: #3124846e;
  color: #0a0141;
  font-size: 26px;
  top: 11%;
  left: 50%;
  transform: translate(-50%, -50%);
  font-family: Arial, Helvetica, sans-serif;
  height: 155px;
  width: 155px;
  display: flex;
  flex-direction: column;
  justify-content: center;
}

.date-time > * {
  text-align: center;
  margin: 5px;
}

.date-time > h2 {
  font-size: 27px;
}

.date-time > h4 {
  font-size: 18px;
}
