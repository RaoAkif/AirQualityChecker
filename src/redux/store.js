import { configureStore } from '@reduxjs/toolkit';
// import thunk from 'redux-thunk';
// import { logger } from 'redux-logger';
import weatherReducer from './weather/weatherSlice';

export default configureStore({
  reducer: weatherReducer,
  // middleware: [thunk, logger],
});
