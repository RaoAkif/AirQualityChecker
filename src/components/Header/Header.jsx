import React from 'react';
import header from '../../assets/images/header.svg';
import './style.css';

function Header() {
  return (
    <div className="header">
      <img src={header} alt="header banner" />
    </div>
  );
}

export default Header;
