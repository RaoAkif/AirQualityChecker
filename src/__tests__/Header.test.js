import renderer from 'react-test-renderer';
import '@testing-library/jest-dom';
import { Provider } from 'react-redux';
import store from '../redux/store';
import Header from '../components/Header/Header';

it('renders correctly', () => {
  const tree = renderer
    .create(
      <Provider store={store}>
        <Header />
      </Provider>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
