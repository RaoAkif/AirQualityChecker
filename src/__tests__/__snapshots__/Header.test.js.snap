// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="header"
>
  <img
    alt="header banner"
    src="header.png"
  />
  <div
    className="date-time"
  >
    <h2>
      10:23:57 PM
    </h2>
    <h4>
      5/19/2022
    </h4>
  </div>
</div>
`;
